src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/InitialConsiderationService.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class InitialConsiderationService {
+    private static final String HORIZONTAL_RULE = "<hr>";
+
+    private static final String completeICTitle = "<h3>What happens next</h3>";
+    private static final String completeICText =
+            "<p>A tribunal caseworker will act on any instructions set out in your initial consideration to progress the case. " +
+                    "You can <a href=\"/cases/case-details/${[CASE_REFERENCE]}#Documents\" target=\"_blank\">view the initial " +
+                    "consideration document in the Documents tab (opens in new tab).</a></p>";
+
+    public String getCompletionText(){
+        return new StringBuilder().append(HORIZONTAL_RULE).append(completeICTitle).append(completeICText).toString();

```

Comment:
Where you're doing lots of appends, it's easier to read split over multiple lines:
```
return new StringBuilder()
.append(HORIZONTAL_RULE)
.append(completeICTitle)
.append(completeICText)
.toString();
```

Also, in this particular case, why have you got this as three separate strings, appending each time the method is called?
Surely you would declare a simple constant that has it all and make it public rather than adding code here for the sake of it, or have I missed something?

Reply:
n/a, deleted class